document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/handler/accounts/SmallFullAccountsDataHandler.java

```diff
@@ -73,30 +81,38 @@ public DocumentInfoResponse getSmallFullAccountsData(Transaction transaction, St
         }
     }
 
-    private CompanyAccounts getCompanyAccounts(String resourceLink, String requestId) throws HandlerException {
-
-        CompanyAccounts accounts;
+    private Transaction getTransaction(CompanyAccounts companyAccounts, String requestId, String resourceId)
+            throws HandlerException {
```

Comment:
This should be in a re-usable class for use by other account data handlers.

Reply:
it is only making a call to the reusable service transactionService.getTransaction.